sklearn/naive_bayes.py

```diff
@@ -785,7 +795,12 @@ class ComplementNB(_BaseDiscreteNB):
     Parameters
     ----------
     alpha : float, default=1.0
-        Additive (Laplace/Lidstone) smoothing parameter (0 for no smoothing).
+        Additive (Laplace/Lidstone) smoothing parameter
+        (set alpha=0 and alphaCorrection=False, for no smoothing).
+
+    alphaCorrection : bool, default=True
```

Comment:
we only use camel case in class names.

Reply:
Should I change name to: alpha_correction or something like that?